Bugsnag/Helpers/BSGDefines.h

```diff
@@ -8,6 +8,28 @@
 #ifndef BSGDefines_h
 #define BSGDefines_h
 
+#include <TargetConditionals.h>
+
 #define BSG_PRIVATE __attribute__((visibility("hidden")))
 
+// Primary capabilities (depends on OS)
+#define BSG_HAVE_APPKIT                       (TARGET_OS_OSX                                                    )
+#define BSG_HAVE_MACH_EXCEPTIONS              (TARGET_OS_OSX || TARGET_OS_IOS                                   )
+#define BSG_HAVE_MACH_THREADS                 (TARGET_OS_OSX || TARGET_OS_IOS || TARGET_OS_TV                   )
+#define BSG_HAVE_OOM_DETECTION                (                 TARGET_OS_IOS || TARGET_OS_TV                   )
+#define BSG_HAVE_REACHABILITY                 (TARGET_OS_OSX || TARGET_OS_IOS || TARGET_OS_TV                   )
+#define BSG_HAVE_REACHABILITY_WWAN            (                 TARGET_OS_IOS || TARGET_OS_TV                   )
+#define BSG_HAVE_SIGNAL                       (TARGET_OS_OSX || TARGET_OS_IOS || TARGET_OS_TV                   )
+#define BSG_HAVE_SIGNALTSTACK                 (TARGET_OS_OSX || TARGET_OS_IOS                                   )
```

Comment:
There's no 'n' in the name of this API

```suggestion
#define BSG_HAVE_SIGALTSTACK                  (TARGET_OS_OSX || TARGET_OS_IOS                                   )
```